requirements/edx/base.txt

```diff
@@ -170,7 +170,7 @@ newrelic==4.20.1.121
 nltk==3.4.4
 nodeenv==1.1.1
 numpy==1.16.4
-oauth2==1.9.0.post1
+git+https://github.com/joestump/python-oauth2.git@master#egg=oauth2
```

Comment:
added the Github version here as a requirement

Reply:
this will also go to github.in